substanced/folder/static/css/mimetype-icons.css

```diff
@@ -0,0 +1,392 @@
+/*
```

Comment:
Techically they could be on either place. I've choosen to put them here, because the folder sub-package is pretty self-contained, and this way it remains self-contained including the static assets it uses.